src/XMakeTasks/Microsoft.Xaml.targets

```diff
@@ -13,7 +13,7 @@
         the .NET Framework directory and thus will no longer be the right answer. Override it to point
         to the correct .NET Framework location. --> 
    <PropertyGroup>
-      <XamlBuildTaskPath Condition="'$(XamlBuildTaskPath)' == ''">$(MSBuildFrameworkToolsPath)</XamlBuildTaskPath>
+      <XamlBuildTaskPath Condition="'$(XamlBuildTaskPath)' == ''">$(MSBuildToolsPath)</XamlBuildTaskPath>
```

Comment:
This doesn't quite make sense. This property would be `$(MSBuildToolsPath)\Microsoft.Xaml.targets` which when deployed and executing is _this_ file. If this file + that DLL is going to come from the VS installer, I don't think there's any change needed to this file. Ideally this could come from a NuGet package so we could enable this scenario outside of VS. But I'm fine if that's not the case for now.